components/cd-dropdown/cd-dropdown.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr" class="no-js">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
```

Comment:
Just curious mainly, but most other HTML files are fragments that don't have `<html>`, `<head>`, `<body>` and so forth. Is this a special case?

Reply:
Yep, this was so I could emulate Drupal's no-js / js class added to the html tag so the cd-dropdown.js in this component could stay as close as possible to the one used in Drupal (`common_design/js/cd-dropdown.js`)
https://github.com/UN-OCHA/common_design/blob/cd-226-design-system/components/cd-dropdown/cd-dropdown.html#L9
https://github.com/UN-OCHA/common_design/blob/cd-226-design-system/components/cd-dropdown/cd-dropdown.js#L384

Reply:
gotcha 👍